lua/lint/linters/eslint_d.lua

```diff
@@ -15,8 +15,12 @@ return require('lint.util').inject_cmd_exe({
     end
     return 'eslint_d'
   end,
-  args = {},
-  stdin = false,
+  args = {
+    '--stdin',
+    '--stdin-filename',
+    vim.api.nvim_buf_get_name(0),
```

Comment:
```suggestion
    function() return vim.api.nvim_buf_get_name(0) end,
```

Otherwise it would freeze the stdin-filename to the name of the buffer it had when `require` is used the first time.